UI/src/components/widgets/performance/view.html

```diff
@@ -1,198 +1,200 @@
+                                    </tr>
+                                    <tr>
+                                        <td>
+                                        </td>
+                                        <td>
+                                            Calls Per Minute
+                                        </td>
+                                        <td>
+                                            {{performanceView.callspm}}
+                                        </td>
+                                    </tr>
+                                </table>
+                            </div>
                         </div>
+                        <!-- End Overall Health -->
                     </div>
-                    <!-- End Overall Health -->
                 </div>
-            </div>
 
-            <div class="col-lg-1 col-sm-2">
-                <!-- Warning -->
-                <div class="widget">
-                    <div class="widget-heading">
-                        Warning
-                    </div>
-                    <div class="widget-body">
-                        <div hide-empty="['{{ performanceView.good[0] }}', '{{ performanceView.warning[0] }}', '{{ performanceView.bad[0] }}']">
+                <div class="col-lg-1 col-sm-2">
+                    <!-- Warning -->
+                    <div class="widget">
+                        <div class="widget-heading">
+                            Warning
+                        </div>
+                        <div class="widget-body">
+                            <div hide-empty="['{{ performanceView.good[0] }}', '{{ performanceView.warning[0] }}', '{{ performanceView.bad[0] }}']">
 
-                            <table class="table violations" style="cursor:pointer;">
-                                <tr>
-                                    <td ng-click="performanceView.showDetail(0)">
-                                        <span dash-status="ok"></span>
-                                        {{performanceView.good.length}}
-                                    </td>
-                                </tr>
-                                <tr>
-                                    <td ng-click="performanceView.showDetail(1)">
-                                        <span dash-status="warning"></span>
-                                        {{performanceView.warning.length}}
-                                    </td>
-                                </tr>
-                                <tr>
-                                    <td ng-click="performanceView.showDetail(2)">
-                                        <span dash-status="alert"></span>
-                                        {{performanceView.bad.length}}
-                                    </td>
-                                </tr>
-                            </table>
+                                <table class="table violations" style="cursor:pointer;">
```

Comment:
@kheinz04  can we get rid of table and move to div

Reply:
consider for future enhancement?